ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml

```diff
@@ -66,6 +66,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: "4.14"
+  source:
```

Comment:
stable ci does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks